flink-table/flink-table-planner/src/main/java/org/apache/calcite/avatica/util/DateTimeUtils.java

```diff
@@ -379,6 +394,29 @@ private static void julianToString(StringBuilder buf, int julian) {
 		int month = m + 3 - 12 * (m / 10);
 		int year = b * 100 + d - 4800 + (m / 10);
 
+//		// this shifts the epoch back to astronomical year -4800 instead of the
+//		// start of the Christian era in year AD 1 of the proleptic Gregorian
+//		// calendar.
+//		int j = julian + 32044;
+//		int g = j / 146097;
+//		int dg = j % 146097;
+//		int c = (dg / 36524 + 1) * 3 / 4;
+//		int dc = dg - c * 36524;
+//		int b = dc / 1461;
+//		int db = dc % 1461;
+//		int a = (db / 365 + 1) * 3 / 4;
+//		int da = db - a * 365;
+//
+//		// integer number of full years elapsed since March 1, 4801 BC
+//		int y = g * 400 + c * 100 + b * 4 + a;
+//		// integer number of full months elapsed since the last March 1
+//		int m = (da * 5 + 308) / 153 - 2;
+//		// number of days elapsed since day 1 of the month
+//		int d = da - (m + 4) * 153 / 5 + 122;
+//		int year = y - 4800 + (m + 2) / 12;
+//		int month = (m + 2) % 12 + 1;
+//		int day = d + 1;
```

Comment:
Useless comments

Reply:
this is explicitly commented out on purpose.